static/css/style.css

```diff
@@ -1116,6 +1116,12 @@ div#container-display-show {
     width: 300px;
 }
 
+#show-selector {
+    width: auto;
+    height:25px;
+    padding:1px;
+}
+
```

Comment:
@p0psicles didn't have the same effect

![image](https://user-images.githubusercontent.com/2620870/29679044-0978f79c-88d7-11e7-876e-5fae46de8690.png)